src/main/scala/io/gatling/sbt/GatlingPlugin.scala

```diff
@@ -94,7 +96,23 @@ object GatlingPlugin extends AutoPlugin {
     config / enterprisePackage := buildEnterprisePackage(config).value,
     config / enterpriseUpload := uploadEnterprisePackage(config).value,
     config / enterprisePackageId := "",
-    config / enterpriseApiToken := sys.props.get("gatling.enterprise.apiToken").orElse(sys.env.get("GATLING_ENTERPRISE_API_TOKEN")).getOrElse("")
+    config / enterpriseApiToken := sys.props.get("gatling.enterprise.apiToken").orElse(sys.env.get("GATLING_ENTERPRISE_API_TOKEN")).getOrElse(""),
+    config / packageBin := (config / enterprisePackage).value // If we directly use config / enterprisePackage for publishing, classifiers (-tests or -it) are not correctly handled.
+  )
+
+  private def gatlingGlobalbaseSettings(config: Configuration) = Seq(
```

Comment:
As this method defines settings on (sub)project
```suggestion
  private def gatlingProjectBaseSettings(config: Configuration) = Seq(
```